docker/1.13.1/Dockerfile.gpu

```diff
@@ -0,0 +1,132 @@
+FROM nvidia/cuda:10.0-base-ubuntu16.04
+
+LABEL maintainer="Amazon AI"
+
+RUN apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated \
+        software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa -y && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated \
+        ca-certificates \
+        cuda-command-line-tools-10-0 \
+        cuda-cublas-dev-10-0 \
+        cuda-cudart-dev-10-0 \
+        cuda-cufft-dev-10-0 \
+        cuda-curand-dev-10-0 \
+        cuda-cusolver-dev-10-0 \
+        cuda-cusparse-dev-10-0 \
+        curl \
+        libcudnn7=7.4.1.5-1+cuda10.0 \
+        # TensorFlow doesn't require libnccl anymore but Open MPI still depends on it
+        libnccl2 \
```

Comment:
As this is the dependencies of OpenMPI, and openmpi is pinned with version, could you please pin this version as well?